packages/logo/src/Logo.tsx

```diff
@@ -37,22 +37,28 @@ interface GenericLogoProps extends LogoProps {
  *
  * Note: For performance, it's recommended to import a specific logo explicitly rather than rely on this generic component
  */
-export default function GenericLogo({
-  name = 'MongoDBLogo',
-  ...rest
-}: GenericLogoProps) {
-  const LogoMap = {
-    MongoDBLogo,
-    MongoDBLogoMark,
-    AtlasNavGraphic,
-    AtlasLogoLockup,
-    AtlasForGovernmentLogoLockup,
-    EnterpriseAdvancedLogoLockup,
-    CommunityEditionLogoLockup,
-    UniversityLogoLockup,
-  };
-
-  const Logo = LogoMap[name];
-
-  return <Logo {...rest} />;
-}
+const GenericLogo = React.forwardRef(
+  (
+    { name = 'MongoDBLogo', ...rest }: GenericLogoProps,
+    ref: React.Ref<SVGSVGElement> | undefined,
+  ) => {
+    const LogoMap = {
+      MongoDBLogo,
+      MongoDBLogoMark,
+      AtlasNavGraphic,
+      AtlasLogoLockup,
+      AtlasForGovernmentLogoLockup,
+      EnterpriseAdvancedLogoLockup,
+      CommunityEditionLogoLockup,
+      UniversityLogoLockup,
+    };
+
+    const Logo = LogoMap[name];
+
+    return <Logo {...rest} ref={ref} />;
+  },
+);
+
+GenericLogo.displayName = 'Logo';
+
+export default GenericLogo;
```

Comment:
While you're making edits, could you change these from default exports to named exports? They're named in the index file, so this will just keep the names consistent

Reply:
Done